services/backend/src/docs/definitions.yml

```diff
@@ -242,6 +242,8 @@ definitions:
         type: boolean
       developmentalGoals:
         type: boolean
+      description:
+        type: Boolean
```

Comment:
I know you've pointed out that the docs are a bit out of date, could you create a new enum (like the ones at the top of this file and make it contain the values of VisibleCards (**PRIVATE**, **PUBLIC**, **CONNECTIONS**) and change all of types to something like `$ref: "#/definitions/CardVisibilityStatusEnum"` (let's say the enum is called `CardVisibilityStatusEnum`)

Reply:
added enum for cardVisibilitys